Snowflake.Data/Core/Session/SFSessionProperty.cs

```diff
@@ -76,6 +76,8 @@ internal enum SFSessionProperty
         GCS_USE_DOWNSCOPED_CREDENTIAL,
         [SFSessionPropertyAttr(required = false, defaultValue = "false")]
         FORCEPARSEERROR,
+        [SFSessionPropertyAttr(required = false, defaultValue = "60")]

```

Comment:
The issue mention 120 seconds. Why we select 60 secs? Is it because other driver also has 60 sec? We need to make sure in a valid case when user can't enter credentials in a minutes or somehow response is not coming back in 60 sec, user is not seeing as regression. So we need to decide what's a good timeout so won't feel like hang and still regular case works fine and user don't need to hurry to punch the credentials.